### ISSUE TITLE
on "shadow" doesn't get triggered if I move the element upwards

### ISSUE THREAD

**Issue by dbertella** [bevacqua/dragula#138]
I'm probably missing something, but I'm trying to move an element inside a div container and I need to get its position while is moving. I'm using .on("shadow") and it's working great moving the element down, left or right but it seems that the event doesn't get triggered if I move my element upwards. Do you may know what am I missing?

**Comment by dbertella** [comment]
I'm trying to debug it in a plunker and I noticed that if I remove this line of css:
.gu-hide {
  display: none !important;
}
my icon it doesn't move anymore in any direction.
Here is the plunker.
http://embed.plnkr.co/Efa74I1on325k7Dhq0me/

**Comment by bevacqua** [comment]
Well sure, you shouldn't be removing that line. That's completely unrelated, though.

**Comment by bevacqua** [comment]
Check if the latest release fixes your issues, but I wasn't able to reproduce your bug report.

**Comment by dbertella** [comment]
Hi Nicolas, thank you very much for looking into it, I've tried with the
new version but nothing change.

Did you have the chance to look into the plunker maybe?
What I'm trying to do is moving a div around a single area, the div (the
blue circle in the example) has position absolute because I needed to let
the user move it around and I set its top and left property on dragend. I
take the coordinate on shadow because is the only event I found triggering
on element moving.
I don't know if this is the right way of using dragula, but it seems
working apart from upwards direction.
If you check the console it prints coordinates on element dragging, and you
can see that if you are trying to go up nothing will be printed.

2015-07-30 2:30 GMT+02:00 Nicolas Bevacqua <notifications@github.com>:

> Check if the latest release fixes your issues, but I wasn't able to
> reproduce your bug report.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bevacqua/dragula/issues/138#issuecomment-126138523>.
>



-- 
Daniele Bertella
*web designer & developer *

+39.340.490.57.80
info@danielebertella.com
LinkedIn: danielebertella <http://it.linkedin.com/in/danielebertella/>
Twitter: _denb <https://twitter.com/_denb>
www.danielebertella.com
